spec/interfaces.md

```diff
@@ -289,759 +319,746 @@ The type of an interface event shall be input-safe.
 
 ### Uniqueness of implemented interfaces
 
-The interfaces implemented by a generic type declaration shall remain unique for all possible constructed types. Without this rule, it would be impossible to determine the correct method to call for certain constructed types. \[*Example*: Suppose a generic class declaration were permitted to be written as follows:
+The interfaces implemented by a generic type declaration shall remain unique for all possible constructed types. Without this rule, it would be impossible to determine the correct method to call for certain constructed types. [*Example*: Suppose a generic class declaration were permitted to be written as follows:
 
-interface I&lt;T&gt;\
-{\
-void F();\
+```csharp
+interface I<T>
+{
+void F();
 }
-
-class X&lt;U,V&gt;: I&lt;U&gt;, I&lt;V&gt; // Error: I&lt;U&gt; and I&lt;V&gt; conflict\
-{\
-void I&lt;U&gt;.F() {…}\
-void I&lt;V&gt;.F() {…}\
+class X<U,V>: I<U>, I<V> // Error: I<U> and I<V> conflict
+{
+    void I<U>.F() {…}
+    void I<V>.F() {…}
 }
+```
 
 Were this permitted, it would be impossible to determine which code to execute in the following case:
 
-I&lt;int&gt; x = new X&lt;int,int&gt;();\
+```csharp
+I<int> x = new X<int,int>();
 x.F();
+```
 
-*end example*\]
+*end example*]
 
 To determine if the interface list of a generic type declaration is valid, the following steps are performed:
 
--   Let L be the list of interfaces directly specified in a generic class, struct, or interface declaration C.
-
--   Add to L any base interfaces of the interfaces already in L.
-
--   Remove any duplicates from L.
-
--   If any possible constructed type created from C would, after type arguments are substituted into L, cause two interfaces in L to be identical, then the declaration of C is invalid. Constraint declarations are not considered when determining all possible constructed types.
+- Let L be the list of interfaces directly specified in a generic class, struct, or interface declaration C.
+- Add to L any base interfaces of the interfaces already in L.
+- Remove any duplicates from L.
+- If any possible constructed type created from C would, after type arguments are substituted into L, cause two interfaces in L to be identical, then the declaration of C is invalid. Constraint declarations are not considered when determining all possible constructed types.
 
-\[*Note*: In the class declaration X above, the interface list L consists of I&lt;U&gt; and I&lt;V&gt;. The declaration is invalid because any constructed type with U and V being the same type would cause these two interfaces to be identical types. *end note*\]
+> [!NOTE] In the class declaration X above, the interface list L consists of l&lt;U&gt; and I&lt;V&gt;. The declaration is invalid because any constructed type with U and V being the same type would cause these two interfaces to be identical types.
```

Comment:
The generics should be code-fenced instead of HTML encoded:

```suggestion
> [!NOTE] In the class declaration `X` above, the interface list `L` consists of `l<U>` and `I<V>`. The declaration is invalid because any constructed type with `U` and `V` being the same type would cause these two interfaces to be identical types.
```